docs/usage/configuration-options.md

```diff
@@ -1679,6 +1684,26 @@ Use this field to restrict rules to a particular language. e.g.
 }
 ```
 
+### matchCategories
+
+Use this field to restrict rules to a particular language or group. The categories can be found on the [manager documentation](./modules/manager.md). e.g.
+
+<!-- prettier-ignore -->
+!!! warning
+    Rules with `matchCategories` are only applied after extraction of a repository, therefore it can not be used to completely prevent analyzes of a repository.
+    Still it can be used to prevent or change the renovation of dependencies.
+
+```json
+{
+  "packageRules": [
+    {
+      "matchCategories": ["python"],
+      "enabled": false
+    }
+  ]
+}
+```
+
```

Comment:
```suggestion

In the above example, this rule will not prevent Renovate from extracting Python dependencies, but it will stop them from being updated.
If you wish to skip extraction altogether, you'd need to use a feature like `enabledManagers` instead.

```